python/ucxx/_lib/tag.pyx

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+from libc.stdint cimport uint64_t
+
+
+cdef class UCXXTag:
+    cdef:
+        readonly uint64_t value
```

Comment:
Do we need this indentation? From the [Cython docs](https://cython.readthedocs.io/en/stable/src/userguide/extension_types.html#static-attributes), I would've thought we could just do


```suggestion
    cdef readonly uint64_t value
```

Reply:
In this case here we don't need it because it's a single variable, it gets more useful when you have multiple variables.

Reply:
That makes sense to me - in that case, I'll leave this up to your preference, just wanted to make sure I wasn't missing something 🙂